src/frontend/components/TeacherLiveLifecycleControls/StopLiveButton/index.spec.tsx

```diff
@@ -11,25 +23,99 @@ jest.mock('data/appData', () => ({
   appData: { jwt: 'cool_token_m8' },
 }));
 
-const mockSetStopLiveConfirmation = jest.fn();
-jest.mock('data/stores/useStopLiveConfirmation', () => ({
-  useStopLiveConfirmation: () => [false, mockSetStopLiveConfirmation],
+jest.mock('data/sideEffects/stopLive', () => ({
+  stopLive: jest.fn(),
 }));
+const mockedStopLive = stopLive as jest.MockedFunction<typeof stopLive>;
+
+let matchMedia: MatchMediaMock;
 
 describe('<StopLiveButton />', () => {
+  beforeEach(() => {
+    matchMedia = new MatchMediaMock();
+  });
   afterEach(() => {
     jest.clearAllMocks();
+    matchMedia.clear();
+
+    /*
+      make sure to remove all body children, grommet layer gets rendered twice, known issue
+      https://github.com/grommet/grommet/issues/5200
+    */
+    document.body.innerHTML = '';
+    document.body.appendChild(document.createElement('div'));
```

Comment:
I discovered testing-library `clear()` in your PR.
Maybe it could replace this ?

Reply:
by `clear()` do you mean `cleanup()` ?
If it is the case, this is not working for layers unfortunatly